test/files/pos/t8111.scala

```diff
@@ -7,10 +7,10 @@ trait T {
     foo((u: Unit) => ma)
     foo(0, (u: Any) => ma) apply ()
 
-    // crash due to side effects on the onwer of the symbol in the 
+    // crash due to side effects on the owner of the symbol in the
     // qualifier or arguments of the application during an abandoned
-    // names/defaults transform. The code type checkes because of
-    // autp-tupling which promotes and empty parmater list to `(): Unit`
+    // names/defaults transform. The code type checks because of
+    // auto-tupling which promotes an empty parameter list to `(): Unit`
```

Comment:
They have standardized on typecheck w/o hyphen, and probably also autotupling, such is the progress of language development, but if autotupling is deprecated, do the natural language progressions still obtain? I'd give these a pass.